### PR TITLE
doc(open meetings): add open meeting for portal

### PR DESCRIPTION
**Pull request by stephanbcbauer**
## Description

This PR adds a new open meeting to our [open meetings](https://eclipse-tractusx.github.io/community/open-meetings/) page.

image

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by wiegerterik** [comment]
Please update the time as requested to start at 1:30pm

**Comment by stephanbcbauer** [comment]
> Please update the time as requested to start at 1:30pm

@wiegerterik thanks. Please check again.

File: community/open-meetings.mdx [hunk 1]
```diff
@@ -58,6 +58,14 @@ These are dedicated sync meetings for specific products, as well as open plannin
              }
 />
 
+<MeetingInfo title="[Portal] Portal Open Meeting"
```
Comment (evegufy) [inline community/open-meetings.mdx:61]: could you please remove the `[Portal]`, it's quite redunant

File: community/open-meetings.mdx [hunk 2]
```diff
@@ -58,6 +58,14 @@ These are dedicated sync meetings for specific products, as well as open plannin
...
+             schedule="Every Thursday effective 4. Jul 2024 until 31. Dec 2024 from 01:30 pm to 02:00 pm CEST"
+             description="Coordination of feature refinement and development for portal."
```
Comment (evegufy) [inline community/open-meetings.mdx:63]: could you please change to Portal product not just portal.

File: community/open-meetings.mdx [hunk 3]
```diff
@@ -58,6 +58,14 @@ These are dedicated sync meetings for specific products, as well as open plannin
...
+             contact="erik.wiegert@cofinity-x.com"
+             sessionLink="https://teams.microsoft.com/l/meetup-join/19%3ameeting_YmIyYjgwOTQtZjI1Ny00YmM0LTlmOWQtODRjZWFmZWM1Y2E3%40thread.v2/0?context=%7b%22Tid%22%3a%221ad22c6d-2f08-4f05-a0ba-e17f6ce88380%22%2c%22Oid%22%3a%22a8b7a5ee-66ff-4695-afa2-08f893d8aaf6%22%7d"
+             meetingLink="/meetings/portal-open-meeting.ics"
+/>
+
```
Comment (evegufy) [inline community/open-meetings.mdx:68]: would you great if we could also link the matrix chat https://chat.eclipse.org/#/room/#tractusx-portal:matrix.eclipse.org

**Review by evegufy — CHANGES_REQUESTED** [review]
nice idea! just some small comments

File: community/open-meetings.mdx (hunk 2 above)
Comment (stephanbcbauer) [reply community/open-meetings.mdx]: adapted. thx

File: community/open-meetings.mdx (hunk 3 above)
Comment (stephanbcbauer) [reply community/open-meetings.mdx:73]: added additional links. thx

**Comment by stephanbcbauer** [comment]
@evegufy thx for your review. New view

image

**Review by ntruchsess — APPROVED** [review]

**Pull request merged**